downloads/install.sh

```diff
@@ -112,6 +112,29 @@ fi
 FILE="spin-${VERSION}-${OS}-${ARC}.tar.gz"
 URL="https://github.com/fermyon/spin/releases/download/${VERSION}/${FILE}"
 
+# Establish the location of current working environment
+current_dir=$(pwd)
+
+# Define a list of items to check for (this can be updated if the spin tarball is updated)
+items=("/spin" "/LICENSE" "/crt.pem" "spin.sig", "/README.md")
+
+# Create a variable that can be called on to question if an exit 1 is required (due to items already existing)
+exit_due_to_existing_item=0
+
+# Iterate through the list of items
+for item in "${items[@]}"; do
+    if [ -e "${current_dir}${item}" ]; then
+        # Update the variable because the found existing item now warrants an exit 1 after this loop
+        exit_due_to_existing_item=1
+        fancy_print 1 "Error ${item} already exists, please delete ${current_dir}${item} and run the installer again."
```

Comment:
Based on the output - it seems to  start with `/` instead of `./`.

```suggestion
        fancy_print 1 "Error .${item} already exists, please delete ${current_dir}${item} and run the installer again."
```

Reply:
Nice catch @karthik2804 I will update in a new push, because that line has other changes in this review.